### PR TITLE
EUI-4766: Content changes as per prescribed wording

### PR DESCRIPTION
**Pull request by Daniel-Lam**
### JIRA link (if applicable) ###
[EUI-4766](https://tools.hmcts.net/jira/browse/EUI-4766)

### Change description ###
Amend screen text as per prescribed wording in the screenshots.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by phillip-whitaker-hmcts — APPROVED** [review]
LGTB+

**Review by rayliangatos — DISMISSED** [review]

File: src/noc/containers/noc-affirmation/noc-affirmation.component.html [hunk 1]
```diff
@@ -4,9 +4,9 @@
     <fieldset class="govuk-fieldset" aria-describedby="sign-affirmation">
       <p>You should tick to 'sign' when:</p>
       <ul class="govuk-list govuk-list--bullet">
-        <li>You’re satisfied that all the information is accurate and matches what is written on the case</li>
-        <li>You have served notice of this change on every party to the claim (including the former legal
-          representative)
+        <li>You're satisfied that all these details are accurate and match what is written on the case</li>
```
Comment (paulhowes-HMCTS) [inline src/noc/containers/noc-affirmation/noc-affirmation.component.html:7]: I see you have kept "You're" in on this line - I assume they didn't want to change this to "You are" like you have done with "you've"

**Review by johnbenjamin-hmcts — APPROVED** [review]
Approved

File: src/noc/containers/noc-affirmation/noc-affirmation.component.html (hunk 1 above)
Comment (Daniel-Lam) [reply src/noc/containers/noc-affirmation/noc-affirmation.component.html:7]: Good spot @paulhowes-HMCTS - however, the above wording was what was in the screenshot and they didn't raise any observations on this. From Jamie Mistry's comments:
> Hi....the wording in the screenshot is the one that has been agreed with Steve C and the rules committee

Comment (paulhowes-HMCTS) [reply src/noc/containers/noc-affirmation/noc-affirmation.component.html:7]: Thanks Daniel. I thought as much. Already approved so resolving this conversation.

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by johnbenjamin-hmcts — APPROVED** [review]
Approved

**Pull request merged**